### ISSUE TITLE
generate more than 1 Server-Timing header

### ISSUE THREAD

**Issue by DanielRuf** [tuupola/server-timing-middleware#27]
It would be good to have it configurable, if one (big) or multiple (short) headers should be created.

https://github.com/tuupola/server-timing-middleware/blob/master/src/ServerTimingMiddleware.php#L133

**Comment by DanielRuf** [comment]
See also https://github.com/tuupola/server-timing-middleware/issues/25#issuecomment-2102056831

**Comment by tuupola** [comment]
Yeah, agreed. Quickly thinking having one flag for choosing between single header and multiple headers would be a clean solution. Having a configurable header size in bytes feels more error prone.
